app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -21,3 +20,5 @@ $govuk-assets-path: '/customs-movements/assets/';
 $govuk-images-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-movements/assets/lib/govuk-frontend/govuk/assets/fonts/";
 @import 'lib/govuk-frontend/govuk/all';
+
+@import 'partials/_step-by-step-navigation';
```

Comment:
Not sure about it, but probably `@import 'lib/govuk-frontend/govuk/all';` must be the last one. @PauloPortugal please take a look, because you moved it to the end

Reply:
It have to be just after path variables

Reply:
It has to be like this. The reason being we are using some features from `govuk` so they have to be defined before our code.
By the way it is another thing we have to live with only because we are using design that is not supported for production.